perun-base/src/main/java/cz/metacentrum/perun/registrar/model/ApplicationForm.java

```diff
@@ -16,7 +20,7 @@ public class ApplicationForm {
 	private boolean automaticApproval;
 	private boolean automaticApprovalExtension;
 	private boolean automaticApprovalEmbedded;
-	private String moduleClassName;
+	private final List<String> moduleClassNames = new ArrayList<>();
```

Comment:
Shouldn't this we a set instead of a list?

Reply:
I'm not too familiar with registrar modules, so this is up to debate with the higher-ups. I was just concerned that if duplicates were in the list, some error might be thrown when the `module.approveApplication` method gets called twice, for example.